src/vip_ipykernel/kernelspec.py

```diff
@@ -0,0 +1,204 @@
+"""Taken and modified from
+https://github.com/ipython/ipykernel/blob/master/ipykernel/kernelspec.py
+
+Copied here to remove the dependency on `ipykernel` itself, as just the kernel
+file needs to be written.
+"""
+
+# Copyright (c) IPython Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+import errno
+import json
+import os
+import shutil
+import sys
+import tempfile
+
+from jupyter_client.kernelspec import KernelSpecManager
+
+pjoin = os.path.join
+
+KERNEL_NAME = 'python%i' % sys.version_info[0]
+
+# path to kernelspec resources
+RESOURCES = pjoin(os.path.dirname(__file__), 'resources')
+
+
+def make_vip_ipkernel_cmd(mod='ipykernel_launcher', executable=None, extra_arguments=None, **kw):
+    """Build Popen command list for launching an ViP-IPython kernel.
+
+    Parameters
+    ----------
+    mod : str, optional (default 'ipykernel_launcher')
+        A string of an IPython module whose __main__ starts an IPython kernel
+
+    executable : str, optional (default sys.executable)
+        The Python executable to use for the kernel process.
+
+    extra_arguments : list, optional
+        A list of extra arguments to pass when executing the launch code.
+
+    Returns
+    -------
+
+    A Popen command list
+    """
+    if executable is None:
+        executable = sys.executable
+    extra_arguments = extra_arguments or []
+    #  When installing the ViP IPykernel, the first `-m` module call points to
+    #  our `vip_ipykernel_launcher`, and the second module call points to the
+    #  desired ipykernel launcher module
+    arguments = [executable, '-m', 'vip_ipykernel.vip_ipykernel_launcher', '-m', mod, '-f', '{connection_file}']
```

Comment:
Here's where I added `'-m', 'vip_ipykernel.vip_ipykernel_launcher'` to 'wrap' the ipykernel launcher call.